src/Multicall3Delegatecall.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {Script} from "forge-std/Script.sol";
```

Comment:
is this dead import?

Reply:
yes it is, good catch.

Reply:
[fafc638](https://github.com/ethereum-optimism/superchain-ops/pull/502/commits/fafc6383103d58514be883c587837f4a6be82f5d)